src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -983,36 +984,40 @@ public partial class JsonStringEnumConverter : System.Text.Json.Serialization.Js
     public enum JsonUnknownDerivedTypeHandling
     {
         FailSerialization = 0,
-        FallbackToBaseType = 1,
-        FallbackToNearestAncestor = 2
+        FallBackToBaseType = 1,
```

Comment:
fallback is a single word so I think proper capitalisation is `Fallback`

Reply:
This is explicitly changing the name based on API review recommendations. We can discuss again today if "fall back" is one word or two words.

Reply:
cc @bartonjs

Reply:
"[fallback](https://en.wiktionary.org/wiki/fallback#English)" is a noun.  "[fall back](https://en.wiktionary.org/wiki/fall_back#English)" is a verb.  Since it's being compared with "fail", a verb, the capital B is more grammatically correct.